Sources/EndPoints/EndPoint.swift

```diff
@@ -32,7 +32,6 @@ extension EndPoint {
 
   var headers: [String: String] { return [:] }
   var encoding: ParameterEncoding { return .json }
-  var method: HTTPMethod { return .get }
```

Comment:
It's better to set up each endpoint specifically because it's not always when we need GET method, so it's potentially leading to problems.